Frameworks/Foundation/NSSet.mm

```diff
@@ -27,8 +27,14 @@
 #include "Starboard.h"
 #include "StubReturn.h"
 #include "VAListHelper.h"
+#import "NSKeyValueObserving-Internal.h"
 #import <_NSKeyValueCodingAggregateFunctions.h>
 
+@interface NSSet ()
+// Public interface for NSSet does not have this method, but is necessary for proper formatting with nested collections
+- (NSString*)descriptionWithLocale:(id)locale indent:(NSUInteger)level;
```

Comment:
You don't really need to declare this; internal consumers can use it because it is in the `@implementation`.